nimble/controller/src/ble_ll_ctrl.c

```diff
@@ -1599,6 +1599,16 @@ ble_ll_ctrl_rx_conn_param_req(struct ble_ll_conn_sm *connsm, uint8_t *dptr,
 {
     uint8_t rsp_opcode;
 
+    /* If Host masked out Remote Connection Parameter Request Event, we need to
+     * send Reject back to the remote device
+     */
+    if (!ble_ll_hci_is_le_event_enabled(BLE_HCI_LE_SUBEV_REM_CONN_PARM_REQ)){
```

Comment:
masked event only matters if event should be indicated to host - you do not know this here. this should be added in `ble_ll_ctrl_conn_param_pdu_proc` - there's fixme comment regarding event masking there.